CMakeModules/CPackConfig.cmake

```diff
@@ -7,30 +7,69 @@
 
 cmake_minimum_required(VERSION 3.5)
 
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/CMakeModules/nsis")
+
 include(Version)
+include(CPackIFW)
 
 set(CPACK_GENERATOR "STGZ;TGZ" CACHE STRINGS "STGZ;TGZ;DEB;RPM;productbuild")
 set_property(CACHE CPACK_GENERATOR PROPERTY STRINGS STGZ DEB RPM productbuild)
 mark_as_advanced(CPACK_GENERATOR)
 
+set(VENDOR_NAME "AccelerEyes LLC")
```

Comment:
ArrayFire LLC

Reply:
We are registered as `AccelerEyes LLC` I believe and old installers also had the same name.